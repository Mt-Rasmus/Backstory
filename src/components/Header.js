import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => ( // props gets passed in, so just destructuring it
   <header className="header">
      <div className="content-container">
         <div className="header__content">
            <Link className="header__title" to="/dashboard">
               <h1>Backstory</h1>
            </Link>
            <div className="header__title">
               <h1>How to Play</h1>
            </div>
         </div>      
      </div>

   </header>
);

export default Header;