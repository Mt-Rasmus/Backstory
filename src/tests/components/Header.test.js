
import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';
//import toJSON from 'enzyme-to-json';

test('Should render Header correctly', () => {
   const wrapper = shallow(<Header />);
   expect(wrapper).toMatchSnapshot();
});
