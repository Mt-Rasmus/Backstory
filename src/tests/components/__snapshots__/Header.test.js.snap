// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <[object Object]
        className="header__title"
        to="/dashboard"
      >
        <h1>
          Backstory
        </h1>
      </[object Object]>
      <h1
        className="header__text"
        onClick={[Function]}
      >
        How to Play
      </h1>
    </div>
    <div>
      <InstructionsModal
        handleToggleModal={[Function]}
        showModal={false}
      />
    </div>
  </div>
</header>
`;
